cpp/include/cudf/types.hpp

```diff
@@ -97,6 +97,14 @@ enum class null_order : bool {
   BEFORE  ///< NULL values ordered *before* all other values
 };
 
+/**---------------------------------------------------------------------------*
+ * @brief Indicates whether the bound of a range is included or excluded.
+ *---------------------------------------------------------------------------**/
+enum class range_bound : bool {
```

Comment:
Sadly Cython doesn't support `enum class`, so we can use `enum class` in external APIs. 
```suggestion
enum range_bound {
```

Reply:
Oh shoot, isn't that going to be an issue with null_order as well then?